source/loader/ze_loader_api.cpp

```diff
@@ -96,48 +96,91 @@ zelLoaderTranslateHandleInternal(
         return ZE_RESULT_SUCCESS;
     }
 
+    bool validHandle = false;
```

Comment:
what about handling invalid handle after the entire `switch`?

Reply:
I am removing the variable because it is not needed and I can simplify the check.